lib/vrl/stdlib/Cargo.toml

```diff
@@ -124,8 +124,9 @@ default = [
     "parse_csv",
     "parse_duration",
     "parse_glog",
-    "parse_grok",
-    "parse_groks",
+# REVISIT syedriko: commented out to avoid dependency on onig-sys which fails to build on f38
```

Comment:
Can we just delete these if not needed? Any reason we would want to bring them back?